resource-aggregate/config.yaml

```diff
@@ -0,0 +1,80 @@
+log:
+  debug: false
+apis:
+  grpc:
+    address: "0.0.0.0:9100"
+    tls:
+      caPool: "/secrets/public/rootca.crt"
+      keyFile: "/secrets/private/cert.key"
+      certFile: "/secrets/public/cert.crt"
+      clientCertificateRequired: true
+    capabilities:
+      snapshotThreshold: 16
+      occMaxRetry: 8
+      userMgmtTickFrequency: 15s
+      userMgmtExpiration: 1m
+clients:
+  eventBus:
+    nats:
+      url:
+      tls:
+        caPool: "/secrets/public/rootca.crt"
+        keyFile: "/secrets/private/cert.key"
+        certFile: "/secrets/public/cert.crt"
+        useSystemCAPool: false
+  eventStore:
+    mongoDB:
+      uri:
+      database: eventStore
+      tls:
+        caPool: "/secrets/public/rootca.crt"
+        keyFile: "/secrets/private/cert.key"
+        certFile: "/secrets/public/cert.crt"
+        useSystemCAPool: false
+      batchSize: 16
+      maxPoolSize: 16
+      maxConnIdleTime: 4m0s
+  oauthProvider:
```

Comment:
resource directory doesn't have oauth providers. It might have oauth client for a specific API.